src/reflect/type.go

```diff
@@ -412,6 +343,7 @@ type Type interface {
 }
 
 // Constants for the 'meta' byte.
+// These constants are also defined in the internal/reflectlite package.
 const (
 	kindMask       = 31  // mask to apply to the meta byte to get the Kind value
```

Comment:
Given `reflectlite` is an internal package, can these constants be shared, rather than duplicated? e.g. `kindMask = reflectlite.KindMask`?

Reply:
@aykevl what do you think? Seems like a good idea to me.

Reply:
Yes, that's definitely possible! I can update this PR if this PR is the one we're going for.